packages/dx-react-grid/src/plugins/table-filter-row.jsx

```diff
@@ -34,6 +34,15 @@ const defaultMessages = {
   lessThanOrEqual: 'Less than or equal to',
 };
 
+const getSelectedFilterOperation = (
+  filterOperations, columnName, filter, columnFilterOperations,
+) => {
+  if (filterOperations[columnName]) {
```

Comment:
Can we extract `getSelectedFilterOperation` function to `helpers` file?

Reply:
It's quite specific and used only here. I'm not sure it's necessary.

Reply:
But in all packages, we extract specific methods to clear plugins code. If we want to support other frameworks we should extract all functionality to common package

Reply:
OK